src/osut/osut.py

```diff
@@ -2555,37 +2555,33 @@ def is_same(s1=None, s2=None, indexed=True) -> bool:
     if not isinstance(indexed, bool): indexed = True
 
     if indexed:
-        xOK = abs(s1[0].x() - s2[0].x()) < CN.TOL
-        yOK = abs(s1[0].y() - s2[0].y()) < CN.TOL
-        zOK = abs(s1[0].z() - s2[0].z()) < CN.TOL
-
-        if xOK and yOK and zOK and len(s1) == 1:
-            return True
+        if len(s1) == 1:
+            if abs(s1[0].x() - s2[0].x()) > CN.TOL: return False
+            if abs(s1[0].y() - s2[0].y()) > CN.TOL: return False
+            if abs(s1[0].z() - s2[0].z()) > CN.TOL: return False
         else:
             indx = None
 
             for i, pt in enumerate(s2):
-                if indx: continue
+                if indx: break
 
-                xOK = abs(s1[0].x() - s2[i].x()) < CN.TOL
-                yOK = abs(s1[0].y() - s2[i].y()) < CN.TOL
-                zOK = abs(s1[0].z() - s2[i].z()) < CN.TOL
+                if abs(s1[0].x() - s2[i].x()) > CN.TOL: continue
+                if abs(s1[0].y() - s2[i].y()) > CN.TOL: continue
+                if abs(s1[0].z() - s2[i].z()) > CN.TOL: continue
 
-                if xOK and yOK and zOK: indx = i
+                indx = i
 
-            if not indx: return False
+            if indx is None: return False
```

Comment:
If a list's index equals 0, `if not indx` returns _False_ - not good! `if indx is None` is better here. Made multiple changes to ensure consistency.